src/Umbraco.Infrastructure/Security/IdentityMapDefinition.cs

```diff
@@ -107,7 +112,8 @@ private void Map(IUser source, BackOfficeIdentityUser target)
             source.GetUserCulture(_textService, _globalSettings).ToString(); // project CultureInfo to string
         target.IsApproved = source.IsApproved;
         target.SecurityStamp = source.SecurityStamp;
-        target.LockoutEnd = source.IsLockedOut ? DateTime.MaxValue.ToUniversalTime() : (DateTime?)null;
+        DateTime? lockedOutUntil = source.LastLockoutDate?.AddMinutes(_securitySettings.UserDefaultLockoutTimeInMinutes);
+        target.LockoutEnd = source.IsLockedOut ? source.LastLockoutDate.HasValue ? lockedOutUntil?.ToUniversalTime() : DateTime.MaxValue.ToUniversalTime() : null;
```

Comment:
Small suggestion to improve readability: since we already computed lockedOutUntil from source.LastLockedOutDate I think we could rewrite this line like this (same remark on the other change below):

`target.LockoutEnd = source.IsLockedOut 
                                  ? (lockedOutUntil ?? DateTime.MaxValue).ToUniversalTime() 
                                  : null;
`
What do you think?

Reply:
Perfect ! 👍

Reply:
Great 🎉
I'll ask HQ on Monday a out the potential breaking changes issue.
Have a  nice week-end!